R/nchar.R

```diff
@@ -1,19 +1,28 @@
 #' Check Number of Characters
-#' 
+#'
 #' Checks the number of characters in the elements of an object.
 #'
 #' @inheritParams params
-#' @param nchar A flag indicating whether x should have characters or a missing value indicating no requirements or a count or count range of the number of characters.
-#' 
+#' @param nchar A flag indicating whether x should have characters or a missing
+#'   value indicating no requirements or a count or count range of the number of
+#'   characters.
+#'
 #' @return An invisible copy of x (if it doesn't throw an error).
 #' @seealso \code{\link{check_pattern}} and \code{\link{check_regex}}
 #' @export
 #'
 #' @examples
 #' check_nchar(c("foo", "bar"), nchar = 3)
-check_nchar <- function(x, nchar = TRUE, x_name = substitute(x),
-                         error = TRUE) {
-  lifecycle::deprecate_soft("0.5.1", "check_nchar()", "chk::chk_match()")
+check_nchar <- function(x, 
+                        nchar = TRUE, 
+                        x_name = substitute(x),
+                        error = TRUE) {
+  lifecycle::deprecate_soft(
+    "0.5.1", 
+    "check_nchar()", 
+    NULL, 
+    "Please use `chk::chk_equal()` with nchar() or `chk::chk_match()` instead."
```

Comment:
Can you change details to 
"Please use `chk::chk_equal()`, `chk::chk_range()`, `chk::chk_subset()` etc with nchar() instead."

Reply:
done